### PR TITLE
[CREATE] 4팀 최종 분석 v0.2

### PR DESCRIPTION
**Pull request by chsop**

### CODE REVIEW

File: 발표준비_onboarding_week2_analysis_v1.2.ipynb [hunk 1]
```diff
@@ -0,0 +1,5771 @@
+{
```
Comment (ttobaegi) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>혹시 freq_cum 변수를 제외하신 이유가 있을까요??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='102'/>

**Review by jaehwan2 — APPROVED** [review]

**Comment by jaehwan2** [comment]
👍👍

File: 발표준비_onboarding_week2_analysis_v1.2.ipynb (hunk 1 above)
Comment (ttobaegi) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>더 특별한 아이디어는 떠올리지 못했고... 유저그룹별로 특정 지역에 집중되어있는 경우에, 할인 이벤트를 진행해도 좋을 것 같아요! </p><pre class="ql-syntax" spellcheck="false">df.query('active=="잠수유저"').groupby('ad1')['count'].sum().sort_values(ascending=False).head(10).plot(kind='bar', rot=0, title='잠수유저 지역 top10')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='125'/>

Comment (ttobaegi) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>불쾌지수와 extreme_weather 변수에 대한 설명이 누락된 것 같은데, 확인 부탁드립니다!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (chsop) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>train+test로 EDA를 해도 되는지, train만 가지고 해야되는지에 대해서 좀 찾아봤는데 분석가마다 의견이 분분한것 같더라구요. (최대한 많은 데이터를 가지고 EDA해서 의미있는 피쳐를 찾는게 낫다 vs. 테스트셋 정보는 EDA에 쓰면 안된다)</p><p>정답이 없는 문제라면 저희가 작업하기 편한 방식으로 하는게 좋을거 같아요..ㅎㅎ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (chsop) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>여기 윗부분 마크다운으로 잘못 들어가있는데 수정이 필요할것 같습니다.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Comment (ttobaegi) [reply 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>넵 이번에 저희는 나눠서 진행하지 않았으니, 이 부분은 생략해도 괜찮지 않을까요?</p>

Comment (Aliasss) [reply 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>EDA를 전체 데이터에 대해서 진행하기도 하고, 처음부터 train/test set 나눈 다음에 EDA하기도 하고 하기 나름이더라구요. 분석 목적에 따라서 다를 것 같아요. </p>

Comment (Aliasss) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>초기 가설 부분 좋은 것 같습니다. 이 가설에 대해서 사전에 저희끼리도 더 얘기를 나눠봤으면 좋았겠네요. 감사합니다!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (Aliasss) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>중요한 부분은 아니지만 round 함수 써서 소수점 둘째~셋째 자리까지만 보여주는 것도 좀 더 깔끔해보일 수 있을 것 같아요 ㅎㅎ</p><p>중요한 건 아니라서 넘어가셔도 됩니다!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='110'/>

Comment (chsop) [inline 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>이부분에 아래 코멘트를 추가해보려고 하는데 어떠실까요!</p><p> </p><p> </p><ul><li><strong>모델 예측 의미</strong></li></ul><p>→ 이용건별 정보가 있는 실전DB 기본 데이터셋에 추가 변수들을 만들고 더해 row 단위로 count값을 예측한 결과</p><p> </p><ul><li><strong>모델 예측 결과</strong></li></ul><p>→ Best model : LightGBM → 평가 metric : MAE 0.049, MSE 0.041</p><p>→ 중요 변수 : user_id, store_id, max_temp(일최고기온) 등</p><p> </p><ul><li><strong>한계 및 제안</strong></li></ul><p>→ 모델 예측에 사용한 store_id, ad1, weekday, goods_type, isholiday 등의 피쳐는 유저가 서비스를 이용함과 동시에 발생하는 데이터로 실무에서 예측 모델의 피쳐로 사용할 수는 없으나, 목표변수인 count값에 어느정도의 상대적 영향력을 가지고 있는지 확인하기 위해 사용하였음</p><p>→ 또한 본 데이터셋에서의 count는 유저가 같은 날짜에 같은 store를 이용한 값으로 대부분이 1의 값을 가지며, count가 2 이상인 건은 유저가 1시간권을 여러장 구매했거나 주차장을 떠났다가 돌아와 주차하는 등의 다양한 시나리오로 발생했을 수 있음</p><p>→ 이렇듯 특이사항이 발생하여 같은 날짜에 같은 store를 여러번 이용한 row 단위 count값을 예측하는 것은 count를 늘리기 위한 비즈니스 전략을 세우는데 큰 도움이 되지 않을 수 있으므로, 유저별 &amp; 월별로 기존의 데이터를 묶어서 한달 이용 건수를 예측하는 모델을 제안</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chsop/team4_wanted_onboarding/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='111'/>

Comment (ttobaegi) [reply 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>아래 추가 분석을 진행한 이유가 잘 설명되어서 좋은 것 같아요! 다만 부정적인 뉘앙스만 조금 바꿔서 작성하면 어떨까요?? </p><p> </p><p>이렇듯 특이사항이 발생하여 <strong>같은 날짜에 같은 store를 여러번 이용한 row 단위 count값을 예측하는 것은 count를 늘리기 위한 비즈니스 전략을 세우는데 큰 도움이 되지 않을 수 있으므로,<em> </em></strong>유저별 &amp; 월별로 기존의 데이터를 묶어서 한달 이용 건수를 예측하는 모델을 제안</p><p> </p><p>→ <strong>유저별 count를 늘리기위한 비즈니스 전략을 제안하고자 유저별&amp;월별 count를 집계하여 향후 이용 건수 예측 모델을 구현하여 추가 분석 진행</strong></p>

Comment (chsop) [reply 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: 넵 반영하겠습니다~!

Comment (chsop) [reply 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: 모델 예측에서는 ranking 컬럼을 다시 살려서 진행중이예요!

Comment (chsop) [reply 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>반영하겠습니다~!</p>

Comment (hwangnakjoo) [reply 발표준비_onboarding_week2_analysis_v1.2.ipynb:1]: <p>이거는 죄송합니다 제가 R에서 쓰는 coord_flip 함수를 python으로 어떻게 쓰는지 한 시간 동안 해봤는데 안 되서 마크다운에 대고 욕좀하느라.. ㅋㅋ</p>
